webrtc-stats.html

```diff
@@ -1860,6 +1879,84 @@ <h2>
           </section>
         </div>
       </section>
+      <section id="mediasourcestats-dict*">
+        <h3>
+          RTCMediaSourceStats dictionary
+        </h3>
+        <p>
+          The <dfn>RTCMediaSourceStats</dfn> dictionary represents a local track that is attached
+          to a sender. It contains information about media sources such as frame rate and
+          resolution pre-encoding. This is in contrast to <code>RTCOutboundRtpStreamStats</code>
+          whose members describe transmission and post-encoding metrics. For example, a track may
+          be captured from a high-resolution camera but be transmitted in low resolution due to
+          CPU and network conditions.
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCMediaSourceStats : RTCStats {
+             unsigned long   width;
+             unsigned long   height;
+             unsigned long   frames;
+             unsigned long   framesPerSecond;
+};</pre>
+          <section>
+            <h2>
+              Dictionary <a class="idlType">RTCMediaSourceStats</a> Members
+            </h2>
+            <dl data-link-for="RTCMediaSourceStats" data-dfn-for="RTCMediaSourceStats"
+            class="dictionary-members">
+              <dt>
+                <dfn><code>width</code></dfn> of type <span class=
+                "idlMemberType">unsigned long</span>
+              </dt>
+              <dd>
+                <p>
+                  Only applicable to video sources. The width, in pixels, of the
+                  last frame originating from this source that was passed to the
+                  sender for further processing. Before the first frame has been
```

Comment:
I just realized one "media source" (track) can be used by multiple senders. I'll update all of these "passed to the sender" to say something like "is ready for further processing". Alternative suggestions are welcomed.

Reply:
I clarified the life cycle at the top and here just said "originating from the source"